AlinaFirsova/Homework 1/Homework_1_Firsova.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE HTML>
+<html>
+ <head>
+  <meta charset="utf-8">
+  <title>Cubes</title>
+<style type="text/css">
+ @import"style.css"
+  </style>
+ </head>
+<body background="img\back.png">
+<div class="center">
+
+<!�������� �������>
+  <table cellspacing="0" cellpadding="0" height="408" width="612">

```

Comment:
@AlinaFirsova Please avoid of using table if it is not representing tabular data.
Look at flex for example